benchmark/portblas/CMakeLists.txt

```diff
@@ -76,12 +76,29 @@ if(${BLAS_ENABLE_EXTENSIONS})
   list(APPEND sources extension/reduction.cpp)
 endif()
 
+# Skip these benchmarks for AdaptiveCpp for SPIRV/OpenCL targets
+# that use SYCL 2020 features like reduction or hang during execution
+# (https://github.com/AdaptiveCpp/AdaptiveCpp/issues/1309)
+set(ADAPTIVE_CPP_SKIP
```

Comment:
Why does this list not include `tbsv`, `trsv` and `tpsv` as mentioned in the `README.md`?

Reply:
those are missing indeed, will ad them!